test/testcafe/spec/IdentifyRegistration_spec.js

```diff
@@ -136,6 +134,82 @@ test.meta('v3', false).requestHooks(mock)('should show errors after empty requir
   await t.expect(registrationPage.hasEmailErrorMessage()).eql(true);
 });
 
+test.requestHooks(mock)('should show max length field validation errors', async t => {
```

Comment:
@trevoring-okta will this behave for non-Latin character, e.g., Chinese, Japanese, Korean (CJK)? Maybe worth testing

https://help.okta.com/en-us/content/topics/reference/ref-supported-languages.htm

Reply:
Good callout, I'm assuming you asked this because some non-Latin characters can be more than one "code unit", which I just learned is what JS `String.length` actually uses to calculate length. I tested gen2 with the 😄 emoji since it takes up 2 code units and it seems that gen2 must use code units to enforce length rather than actual characters because the name fields generated by the mock with `"minLength": 2` allowed a single emoji as a first name.

@lesterchoi-okta Do we want to go with parity for gen2, which would mean leaving the usage of `String.length` as the validation logic, or would you consider this a bug and we should switch to logic that always returns the exact # of characters?

This link has an example for getting the true # of characters: [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/length](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/length)